### PR TITLE
Adding Jsonnet Dependencies to fetch supported  storage mixins

### LINKED ISSUE #13: Add Jsonnet Dependencies to fetch ceph-mixins
ceph-mixins should be fetched via Jsonnet Bundler at build time to get the latest rule file(s).

### PR DESCRIPTION
**Pull request by umangachapagain**
Fixes: /issues/13

Signed-off-by: Umanga Chapagain <chapagainumanga@gmail.com>

### CODE REVIEW

**Comment by umangachapagain** [comment]
@cloudbehl @shtripat @aruniiird  Please review.

**Review by cloudbehl — APPROVED** [review]

File: jsonnet/manifests/prometheus-rules.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apiVersion: monitoring.coreos.com/v1
```
Comment (shtripat) [inline jsonnet/manifests/prometheus-rules.yaml]: I feel this is the file which is generated and checked in as part of every PR. Right?

File: jsonnet/kube-prometheus.libsonnet [hunk 2]
```diff
@@ -0,0 +1,20 @@
+local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
+local configMapList = k.core.v1.configMapList;
+
+(import 'prometheus.libsonnet') +
+(import 'ceph-mixins/mixin.libsonnet') + {
```
Comment (shtripat) [inline jsonnet/kube-prometheus.libsonnet]: How this import gets resolved?

File: jsonnet/manifests/prometheus-rules.yaml (hunk 1 above)
Comment (umangachapagain) [reply jsonnet/manifests/prometheus-rules.yaml]: No. This is what will be generated here during build.

File: jsonnet/kube-prometheus.libsonnet (hunk 2 above)
Comment (umangachapagain) [reply jsonnet/kube-prometheus.libsonnet]: we pull all the dependencies to local (i.e jsonnet/vendor)

**Comment by umangachapagain** [comment]
Changing things to pull multiple projects for build. Few more things needs to be added.

**Pull request merged**